### PR TITLE
Add ToggleToReadme

### PR DESCRIPTION
**Pull request by talum**
Enables batch updating curriculum to remove lab-specific stuff

@ipc103 @jjseymour

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,27 @@
-# Learn Open-Sourcing Script
+# Learn Curriculum Batch Update Script
+
+## Usage
+This script enables the batch updating of curriculum. To use, gather a list
+of repos you want to update and put them in a spreadsheet. Grab the
+absolute path to the CSV.
+
+From the command line, run `bin/run STRATEGY path/to/csv`, where `STRATEGY`
```
Comment (ipc103) [inline README.md:8]: nice.

File: lib/fixtures/LICENSE.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 #Learn.co Educational Content License
 
-Copyright (c) 2015 Flatiron School, Inc
```
Comment (ipc103) [inline lib/fixtures/LICENSE.md:3]: does the copyright refer to when the labs are created, or when the license was created?

File: bin/run [hunk 3]
```diff
@@ -5,11 +5,20 @@ require_relative "../config/environment"
...
+CSV.foreach(csv_path) do |row|
+  repos << Repo.new(row.first)
+end
 
-# write content for each repo
 repos.each do |repo|
-  RepoWriter.new(repo, secrets).write_to_repo
-end
\ No newline at end of file
+  Object.const_get(strategy).new(repo, secrets).execute
```
Comment (ipc103) [inline bin/run:22]: sick

**Review by ipc103 — APPROVED** [review]
Looks great to me. As more use cases come up, now we can simply add more strategies into this script. Nice work!

File: lib/fixtures/LICENSE.md (hunk 2 above)
Comment (talum) [reply lib/fixtures/LICENSE.md:3]: Hm, I'm not actually sure. That's a good question.  This suggests that it doesn't actually matter...http://www.contentious.com/2007/01/07/copyright-notice-is-the-year-really-necessary/.

**Pull request merged**